src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/activities/ActivitiesService.kt

```diff
@@ -174,14 +174,18 @@ class ActivitiesService(
   }
 }
 
-data class OutboundHMPPSDomainEvent(
+data class ScheduleDomainEvent(
   val eventType: String,
-  val identifier: Long,
+  val additionalInformation: ScheduleAdditionalInformation,
   val version: String,
   val description: String,
   val occurredAt: LocalDateTime,
 )
 
+data class ScheduleAdditionalInformation(
+  val activityScheduleId: Long,
+)
+
```

Comment:
The message structure should align with the [HMPPS Domain Event Schema](https://dsdmoj.atlassian.net/wiki/spaces/NDSS/pages/3732570166/ADR+-+Domain+Event+Schema) (and the same change has been made in Activities). And a rename.